nextflow.config

```diff
@@ -26,8 +25,6 @@ params {
 
   minDepth = 10
 
-  clades = "https://github.com/czbiohub/test-datasets/raw/olgabot/msspe--add-human-and-kraken/testdata/data/clades.tsv"
```

Comment:
Delete `assignclades.py` as well, and remove the `--clades` argument from `alignment_assembly_stats.py`